apollo-router/src/plugins/telemetry/tracing/apollo_telemetry.rs

```diff
@@ -333,7 +333,7 @@ impl SpanExporter for Exporter {
         /*
          * Process the batch
          */
-        for span in batch.iter().filter(|span| span.name == "graphql_request") {
+        for span in batch.iter().filter(|span| span.name == "router") {
```

Comment:
I wonder if we should put this span name in a constant to avoid future issues

Reply:
I'm currently working on this part of the code, expect this to not be required anymore soon ™️

Reply:
Fixed in [d802a11](https://github.com/apollographql/router/pull/850/commits/d802a113c5bfb26ba6d4a3af4319dae89d3d331e)